scipy/stats/stats.py

```diff
@@ -3460,15 +3460,16 @@ def pointbiserialr(x, y):
 KendalltauResult = namedtuple('KendalltauResult', ('correlation', 'pvalue'))
 
 
-def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate', method='auto'):
+def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate', method='auto',
+               variant='taub'):
     """
     Calculate Kendall's tau, a correlation measure for ordinal data.
 
     Kendall's tau is a measure of the correspondence between two rankings.
     Values close to 1 indicate strong agreement, values close to -1 indicate
-    strong disagreement.  This is the 1945 "tau-b" version of Kendall's
-    tau [2]_, which can account for ties and which reduces to the 1938 "tau-a"
-    version [1]_ in absence of ties.
+    strong disagreement. This implements all three common variants of Kendall's
+    tau: Tau-a, Tau-b and Tau-c. The default option is tau-b which takes into
```

Comment:
Inconsistent use of Tau and tau

Reply:
What do you think of :math:`tau` throughout?